sysutils/flashrom/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                flashrom
+version             1.0
+categories          sysutils
+platforms           darwin
+license             GPL-2
+maintainers         ra1nb0w openmaintainer
+description         universal flash programming utility
+long_description    utility for identifying, reading, writing, verifying and erasing flash chips.\
+                    It is designed to flash BIOS/EFI/coreboot/firmware/optionROM images on \
+                    mainboards, network/graphics/storage controller cards, and various other \
+                    programmer devices.
+homepage            https://www.flashrom.org/Flashrom
+master_sites        https://download.flashrom.org/releases/
+use_bzip2           yes
+
+checksums           rmd160  b2e0c62d8a0ae592a6d84b69c5f5ea3283dcfefe \
+                    sha256  3702fa215ba5fb5af8e54c852d239899cfa1389194c1e51cb2a170c4dc9dee64 \
+                    size    321693
+
+# needed until next version
+patchfiles          patch-libusb_set_debug-deprecated.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:libftdi0 \
+                    port:libusb-compat \
+                    port:libusb
```

Comment:
Why are both of these libusb dependencies being added? It was my understanding that they are just different versions of the same library. So shouldn't libusb-compat be removed?

Reply:
libusb-compat is a wrapper of libusb to provide compatibility with 0.1 version (used by flashrom). Now I see that libusb-compat has libusb as lib dependency. So I remove libusb. Thanks for checking.